packages/core/src/components/graphs/line.ts

```diff
@@ -44,7 +44,14 @@ export class Line extends Component {
 		this.lineGenerator = line()
 			.x((d, i) => this.services.cartesianScales.getDomainValue(d, i))
 			.y((d, i) => this.services.cartesianScales.getRangeValue(d, i))
-			.curve(this.services.curves.getD3Curve());
+			.curve(this.services.curves.getD3Curve())
+      .defined((d: any, i) => {
```

Comment:
some weird indentation here, is that intentional?
```suggestion
			.defined((d: any, i) => {
```

^ that _may_ fix it

Reply:
it's the github conflict resolver

Reply:
Usually github only barfs like this when it's mixed tabs and spaces, but if you've checked and it's not then it's fine